membership_file/admin.py

```diff
@@ -43,12 +43,12 @@ def get_url(self, obj):
 # Ensures that the last_updated_by field is also updated properly from the Django admin panel
 class MemberWithLog(HideRelatedNameAdmin):
 
-    list_display = ('id', 'first_name', 'tussenvoegsel', 'last_name', 'educational_institution', 'email', 'phone_number', 'marked_for_deletion')
-    list_filter = ['marked_for_deletion', 'educational_institution']
-    list_display_links = ('id', 'first_name')
+    list_display = ('id', 'user', 'first_name', 'tussenvoegsel', 'last_name', 'educational_institution', 'is_deregistered', 'marked_for_deletion')
```

Comment:
why are e-mail and phone number removed?

Reply:
They took quite a bit of screen space, but did not really provide a useful sort or filter option.
They can still be viewed (or changed) on an individual member's page, so they're not removed entirely.